tutorials/pytest.md

```diff
@@ -0,0 +1,220 @@
+# Get Started with Pytest
+
+Pytest jest framework-iem, który ułatwia pisanie testów jednostkowych w Pythonie. Pomaga to osiągać *minimalnym* kodem. Jest również świetnym narzędziem do rozpoczęcia nauki testowania funkcji.
+
+## Intro
+
+"*Test jednostkowy – metoda testowania tworzonego oprogramowania poprzez wykonywanie testów weryfikujących poprawność działania pojedynczych elementów programu – np. metod lub obiektów w programowaniu obiektowym lub procedur w programowaniu proceduralnym.*" [Wikipedia](https://pl.wikipedia.org/wiki/Test_jednostkowy)
+
+Na wstępie należy podkreślić dwa fakty:
+1. Testy powstały z myślą o **aplikacjach**, co oznacza, że mają one testować **metody** z danych **klas** np. czy zwracają odpowiedni format, wymiar, czy nie ma nulli itp. Jest to najwygodniejsza forma to rozbudowanych aplikacji.
+
+> Ponieważ w naszym projekcie będziemy głównie wykorzystywać notatniki, testy będą używane wciąż poprawnie, ale nie do końca "*zgodnie ze sztuką*".
```

Comment:
To nie tak. Testować będziemy wyłącznie elementy z lib, a w tym te użyte potem w notatnikach.

Reply:
Zmieniono